### PR TITLE
Add example using Avaloni.Build.Tasks and path validation

### PR DESCRIPTION
**Pull request by rstm-sf**
## What does the pull request do?
Add example using `Avaloni.Build.Tasks` and path validation. Maybe it could be done better?

## What is the current behavior?
It is not immediately clear what needs to be passed as an argument

## What is the updated/expected behavior with this PR?
Modified output message and moved to stdout. There was also a little refactoring of the code

## How was the solution implemented (if it's not obvious)?
```sh
$ dotnet ./Avalonia.Build.Tasks.dll
```

## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Avaloniaui.net with user documentation

### CODE REVIEW

**Comment by grokys** [comment]
@kekekeks ping

File: src/Avalonia.Build.Tasks/Program.cs [hunk 1]
```diff
@@ -1,70 +1,64 @@
...
+                const string referencesOutputPath = "path/to/Avalonia/samples/Sandbox/obj/Debug/netcoreapp3.1/Avalonia";
+                Console.WriteLine(@$"Usage:
+    1) dotnet ./Avalonia.Build.Tasks.dll <ReferencesOutputPath>
+       , where <ReferencesOutputPath> likes {referencesOutputPath}
+    2) dotnet ./Avalonia.Build.Tasks.dll <AssemblyFilePath> <ReferencesFilePath> <OutputPath>
+       , where:
+           - <AssemblyFilePath> likes {referencesOutputPath}/{_originalDll}
+           - <ReferencesFilePath> likes {referencesOutputPath}/{_references}
+           - <OutputPath> likes {referencesOutputPath}/{_outDll}");
+                return 1;
```
Comment (kekekeks) [inline src/Avalonia.Build.Tasks/Program.cs:26]: It looks like that lines 17-26 are the only relevant changes in this PR. Do other parts of the diff have meaning other than moving code around?

Comment (rstm-sf) [reply src/Avalonia.Build.Tasks/Program.cs]: Left this

Comment (kekekeks) [reply src/Avalonia.Build.Tasks/Program.cs]: Thanks

**Review by kekekeks — APPROVED** [review]

**Pull request merged**